src/upload.ts

```diff
@@ -257,23 +264,24 @@ export async function uploadLargeFileRequest(
   let splitSizeIntoParts:
     | ((totalSize: number, partCount: number) => Array<{ start: number; end: number }>)
     | undefined = undefined;
+  const chunkSize = TUS_CHUNK_SIZE * opts.chunkSizeMultiplier;
```

Comment:
seems like we never validate whether the `chunkSizeMultiplier` is a value greater than zero, same for `parallelUploads`, shouldn't we?

Reply:
Oh yeah, I meant to do that. Good catch.